test/shopify-cli/tasks/update_dashboard_urls_test.rb

```diff
@@ -240,6 +241,8 @@ def test_whitelist_urls_are_updated_even_if_app_url_is_set
           url: 'https://newone123.ngrok.io',
           callback_url: '/callback/fake',
         )
+        assert_requested(get_request)
+        assert_requested(update_request)
```

Comment:
The first passes, the second fails, meaning the stub wasn't triggered. I'm still figuring out why.